src/main/java/com/dataloom/edm/type/Analyzer.java

```diff
@@ -0,0 +1,6 @@
+package com.dataloom.edm.type;
+
+public enum Analyzer {
+    METAPHONE,
+    DEFAULT
```

Comment:
the default elasticsearch analyzer is called "standard", so it might make more sense to change DEFAULT to STANDARD